api/applications/applications.proto

```diff
@@ -9,6 +9,8 @@ import "google/api/annotations.proto";
 
 import "protoc-gen-openapiv2/options/annotations.proto";
 
+import "api/applications/commits.proto";
```

Comment:
Sorry if this wasn't clear. Move all the stuff from `commits.proto` into this file please. As it is here, this is creating a second "service" with its own client. We don't want users to have to use two different client libraries to interact with the Applications service.

Reply:
This actually only has one service that handles both. I added commits under the application service.

Reply:
On second look, you are correct. But we show commits in the context of an application. `/v1/commits` is a bit weird since we don't have a top-level `commit` object, like we do for `application`.

Can you make a case for why they should be separate files?

Reply:
Other than its just cleaner to read and maintain there isnt really another reason why. Ohhhh did you want the url to be v1/application/<appname>/commits because that makes way more sense than the way i did it.

Reply:
> Other than its just cleaner to read and maintain

I would argue one file is easier to read and maintain that two, especially since a lot of boilerplate is repeated between them.

> Ohhhh did you want the url to be v1/application/appname/commits because that makes way more sense than the way i did it.

Yes please. URLs are a little weird for us since we aren't using REST ourselves, and we supply a swagger file for auto-generated clients, but I think the `/application/:name/commits` is pretty sane/predictable.